lib/CPAN/Meta/Prereqs.pm

```diff
@@ -15,6 +15,7 @@ organized by phase and type, as described in L<CPAN::Meta::Prereqs>.
 
 use Carp qw(confess);
 use Scalar::Util qw(blessed);
+use List::Util 1.45 qw(uniq);
```

Comment:
I don't think requiring a recent XS module is a good idea given this module's position in the toolchain.